packages/core/aqua-to-js/src/generate/__test__/__snapshots__/generate.snap.d.ts

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable */
```

Comment:
maybe .gitignore generated code?

Reply:
It's not generated code. These files using in tests. Regenerating them every time will render snapshot testing useless